torch/fx/experimental/symbolic_shapes.py

```diff
@@ -84,6 +92,7 @@ class GuardOnDataDependentSymNode(RuntimeError):
     "is_concrete_bool", "is_singleton", "SHAPEENV_EVENT_KEY", "CURRENT_NODE_KEY",
     "has_free_symbols", "sym_eq", "SymbolicContext", "StatelessSymbolicContext",
     "StatefulSymbolicContext", "SubclassSymbolicContext", "statically_known_true",
+    "guard_size_oblivious",
```

Comment:
Still feels a bit premature to add this as a public doc given potential churn

Reply:
Well, I refuse to give this a leading underscore, and Alban refuses to let me add it to doc ignores, so I suppose we are at am impasse :P